index.html

```diff
@@ -1317,6 +1361,34 @@
     <a>Web NFC message</a>s either from an <a>NFC tag</a> or an <a>NFC peer</a>.
   </p>
 
+  <section> <h3>The <strong>NFCPushTarget</strong> enum</h3>
+    <dl title="enum NFCPushTarget" class="idl">
+      <dt>tag</dt>
+      <dt>peer</dt>
+      <dt>any</dt>
+    </dl>
+  </section>
+
+  <section> <h3>The <strong>NFCPushOptions</strong> dictionary</h3>
+    <dl title="dictionary NFCPushOptions" class="idl">
```

Comment:
+1 on contiguous IDL, but I suggest we do not rush it in this release. IIRC it was @jyasskin's feature :-)